# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../epdfview/01_browser_command.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- epdfview-0.1.7/src/Config.cxx.orig	2006-06-12 21:23:34.000000000 +0200
+++ epdfview-0.1.7/src/Config.cxx	2006-06-23 16:42:19.000000000 +0200
@@ -22,7 +22,7 @@
 using namespace ePDFView;
 
 // Constants
-static const gchar *DEFAULT_EXTERNAL_BROWSER_COMMAND_LINE = "firefox %s";
+static const gchar *DEFAULT_EXTERNAL_BROWSER_COMMAND_LINE = "x-www-browser %s";
 static const gchar *DEFAULT_OPEN_FILE_FOLDER = NULL;
 static const gchar *DEFAULT_SAVE_FILE_FOLDER = NULL;
 static const gboolean DEFAULT_SHOW_STATUSBAR = TRUE;
